xarray/tests/test_backends.py

```diff
@@ -3706,6 +3706,29 @@ def test_save_mfdataset_pathlib_roundtrip(self) -> None:
                 ) as actual:
                     assert_identical(actual, original)
 
+    def test_save_mfdataset_pass_kwargs():
```

Comment:
```suggestion
    def test_save_mfdataset_pass_kwargs(self):
```

Reply:
ah, okay, I see.  Thanks! (I was writing at the same time as you, it seems.)